helm-chart/renku-notebooks/templates/deployment.yaml

```diff
@@ -59,11 +61,15 @@ spec:
             - name: IMAGE_BUILD_TIMEOUT
               value: "{{ .Values.imageBuildTimeout }}"
             - name: IMAGE_REGISTRY
-              value: {{required "An image registry must be specified." .Values.imageRegistry }}
+              value: {{ required "An image registry must be specified." .Values.gitlab.registry.host }}
           ports:
             - name: http
               containerPort: 8000
               protocol: TCP
+          volumeMounts:
+            - name: server-options
+              mountPath: /etc/renku-notebooks/server_options.json
```

Comment:
`/etc/renku-notebooks`
(https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#add-configmap-data-to-a-volume)